### PR TITLE
Add GitVersion stage to Azure Pipelines CI job for SonarCloud and fix code coverage

### PR DESCRIPTION
**Pull request by EraYaN**
Alright now we need to settle on a format.

These are the variables available: https://gitversion.net/docs/more-info/variables

Maybe the hash is not needed for sonarcloud, since it will timestamp them.

### CODE REVIEW

File: .ci/azure-pipelines.yml [hunk 1]
```diff
@@ -15,6 +15,29 @@ trigger:
...
+      - task: UseGitVersion@5
+        inputs:
+          versionSpec: '5.x'
+          includePrerelease: true
+          #targetPath: 'wording dir'
+          #useConfigFile: true
+          #configFilePath: 'config_file.yml'
+          #updateAssemblyInfo: true
+          #updateAssemblyInfoFilename: 'SharedVersion.cs'
+          #additionalArguments: '--additional args'
```
Comment (dkanada) [inline .ci/azure-pipelines.yml:31]:
```suggestion

```

File: Jellyfin.Api/Jellyfin.Api.csproj [hunk 2]
```diff
@@ -7,10 +7,15 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```
Comment (dkanada) [inline Jellyfin.Api/Jellyfin.Api.csproj:10]: Why are you only doing it for the API project?

Comment (EraYaN) [reply Jellyfin.Api/Jellyfin.Api.csproj:10]: It's turned on for every project, this one when it was created did not have this properly done, breaking the versioning. (it was 1.0.0.0)

**Comment by dkanada** [comment]
@joshuaboniface is this still required?

**Comment by EraYaN** [comment]
I don't think so since we now have BUILD_ID versions for the unstable ones. And nightlies will no longer be build.

**Comment by dkanada** [comment]
whichever sounds fine

**Comment by dkanada** [comment]
Those two items don't sound like blockers for 10.6.0 so I'll remove this from the project.

**Comment by oddstr13** [comment]
As for version format, I personally would like a short version of `$InformationalVersion` - something along the lines of `$FullSemVer.$ShortSha.$BranchName`
`3.0.0-beta.1+1.28c8531.release/3.0.0`
`10.6.0+0.c853970.release-10.6.z`

**Comment by dkanada** [comment]
I think the commit hash is more than enough personally.

**Comment by EraYaN** [comment]
This is no longer relevant

**Pull request closed without merging**